### ISSUE TITLE
Setup browser compatibility checks

### ISSUE THREAD

**Issue by twschiller** [twschiller/open-synthesis#107]
Setup browser compatibility checks so that they can be run on a regular basis against the main content.

Possible tools:
- http://browsershots.org/: doesn't like the robots.txt file for some reason
- https://www.keycdn.com/blog/browser-compatibility-testing-tools/

**Comment by HimanshuJanbandhu** [comment]
Hey, I would like to contribute to this, can I take  this issue

**Comment by twschiller** [comment]
Great! I think the easiest thing to do would be to set up a tool like this: https://github.com/amilajack/eslint-plugin-compat

**Comment by HimanshuJanbandhu** [comment]
Thanks Bro! On it...

**Comment by HimanshuJanbandhu** [comment]
Hey, when I am trying to setup the project, it is showing error, 
 > 
from celery import Celery
ModuleNotFoundError: No module named 'celery'
>

**Comment by twschiller** [comment]
Are you following the instructions in `CONTRIBUTING.md`? Here's a couple things that you should check:

- Did you run `pip install -r requirements.txt`? You might try running it again and reviewing the logs to see if there's any error messages related to celery
- The instructions have you set up a Python virtualenv for dependencies. Did you activate the virtualenv? `source venv/bin/activate`. You might be getting the message because it's looking in your global install that doesn't have celery
